content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -116,22 +118,26 @@ There are a few limitations with `env` at the moment:
 
 ### Using .env files
 
-For many developer tools, it has become a good practice to define environment variables using [`.env`](https://github.com/motdotla/dotenv) files.
+For many developer tools, it has become a good practice to define environment variables using [`.env`](https://github.com/motdotla/dotenv) files. Prisma provides support for `.env` files if:
 
-Prisma provides native support for `.env` files **if the `.env` file is located in the same directory as your Prisma schema file**:
+- The `.env` file is located in the same directory as your Prisma schema file
+- You use the `--schema` option to specify the location of the `.env` file
```

Comment:
Both lines here say the same thing: The .env file needs to be next to your schema. By default that is `prisma/schema.prisma`, but can be overridden to whatever you want with `--schema`. In both cases the .env file needs to be next to your schema file.

(The second line is technically not correct - with `--schema` you specify the location of the .env file, which then triggers the CLI to look for the .env file there as well)

Reply:
Maybe the right thing to write here is:
1. Next to your schema file
2. Fallback to `prisma/.env` if not found next to schema

Reply:
I agree, the second line is not correct, as you can't provide the .env file with `--schema`.